R/Connect.R

```diff
@@ -199,6 +196,12 @@ connect <- function(connectionDetails = NULL,
                     oracleDriver = "thin",
                     connectionString = NULL,
                     pathToDriver = getOption("pathToDriver")) {
+  
+  if (!missing(schema) && !is.null(schema)) {
+    warning(
+      "The schema argument is deprecated. Please use a fully specified SQL statement with @databaseSchema."
```

Comment:
I'm not sure everyone will understand. Perhaps: "The schema argument is deprecated. Please use fully specified table names in your SQL statement, for example 'SELECT * FROM my_schema.my_table;'"